src/app/pages/StoryPage/index.jsx

```diff
@@ -249,10 +254,10 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
           parentColumns={gridColumns}
         >
           <ResponsiveComponentWrapper>
-            <TopStories />
+            <TopStories content={topStoriesInitialData} />
           </ResponsiveComponentWrapper>
           <ResponsiveComponentWrapper>
-            <FeaturesAnalysis />
+            <FeaturesAnalysis content={featuresInitialData} />
```

Comment:
now would be a good time to remove the default `content` prop declaration from the [TopStories](https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsTopStories/index.jsx#L45) and [FeaturesAnalysis](https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsFeaturesAnalysis/index.jsx#L45) components

Reply:
I think we may still want the default content prop for now as real fixture data hasn't been added for all services yet.

Reply:
We might just have to create an issue to track removing it if that's the case. But I think some of the services might not have most read and features analysis data and if this is still there we will just be displaying mock content then

Reply:
Yep, what I'll do is create an issue which removes this and also adds fixture data for each service.